daprdocs/content/en/reference/api/metadata_api.md

```diff
@@ -0,0 +1,191 @@
+---
+type: docs
+title: "Metadata API reference"
+linkTitle: "Metadata API"
+description: "Detailed documentation on the Metadata API"
+weight: 800
+---
+
+Dapr has a metadata API that returns information about the sidecar allowing runtime discoverability. The metadata endpoint returns among other things, a list of the components loaded and the activated actors (if present).
+
+The Dapr metadata API also allows you to store additional information in they format of key-value pairs. 
+
+Note: The Dapr metatada endpoint is for instance being used by the dapr cli when running dapr in standalone mode to store the PID of the process hosting the sidecar and the command used to run the application.
```

Comment:
Dapr CLI

Reply:
Thanks for the review. I have just updated the PR to address your comments.